### PR TITLE
Debugger: Format changes only

### PR DESCRIPTION
**Pull request by kiyolee**
- Fix some inconsistent indentations
- Add some curly brackets to clarify structure

### CODE REVIEW

**Comment by Michaelangel007** [comment]
In the future please run all formatting changes for the debugger by me. Some of these are fine, some are not.

Also, smaller PRs are easily to review & accept.

Thx.

**Comment by kiyolee** [comment]
Please just point out which parts you don't like, I can easily fix that.
I guess another common divisive style choice is whether to have extra indentation in front of case tags and how each case block should indent.
I have no interest of pushing personal perference around.
I isolated all these format changes out is mainly to avoid mixing these up with upcoming PRs and so those PRs hopefully can be easier to read.
Some of these format changes were done to help myself to understand the code better while I am making other changes.
I am equally happy if this PR is completely rejected and then I will avoid touching any of these format in upcoming PRs.

File: source/Debugger/Debug.cpp [hunk 1]
```diff
@@ -5252,7 +5251,7 @@ int CmdTextSave (int nArgs)
 	if (g_aArgs[1].bType & TYPE_QUOTED_2)
 		bHaveFileName = true;
 
-	char  *pText;
+	char *pText;
```
Comment (Michaelangel007) [inline source/Debugger/Debug.cpp:5254]: This should be reverted as pText and nSize are aligned. Thanks.

File: source/Debugger/Debug.cpp [hunk 2]
```diff
@@ -5952,84 +5951,84 @@ Update_t CmdOutputPrintf (int nArgs)
...
-								eThis = PS_ESCAPE;
-								break;
-							case '%':
-								eThis = PS_TYPE;
-								break;
-							default:
-								sText[ nLen++ ] = c;
-								break;
-						}
+				case PS_LITERAL:
```
Comment (Michaelangel007) [inline source/Debugger/Debug.cpp:5954]: These case statements needs to be reverted (should be indented.)

File: source/Debugger/Debug.cpp [hunk 3]
```diff
@@ -8659,7 +8658,7 @@ void DebugInitialize ()
 	}
 	
 #if _DEBUG
-//g_bConsoleBufferPaused = true;
+	//g_bConsoleBufferPaused = true;
```
Comment (Michaelangel007) [inline source/Debugger/Debug.cpp:8661]: Another good catch. Thx!

File: source/Debugger/Debugger_Display.cpp [hunk 4]
```diff
@@ -3192,7 +3192,7 @@ void DrawSubWindow_Data (Update_t bUpdate)
 
 		nAddress += nMaxOpcodes;
 
-		rect.top    += nFontHeight;
+		rect.top += nFontHeight;
```
Comment (Michaelangel007) [inline source/Debugger/Debugger_Display.cpp:3195]: This is another good catch. Thx.

**Review by Michaelangel007 — CHANGES_REQUESTED** [review]

**Comment by Michaelangel007** [comment]
OK, I've finished the review.  Overall this has some good catches and cleanup that is long over due.

There is some minor cleanup before I'll can accept this PR:

* case statements need to be reverted and indented
* misc. type / variable alignment

Thanks for doing the grunt work of cleanup.

File: source/Debugger/Debugger_Disassembler.cpp [hunk 5]
```diff
@@ -553,38 +552,33 @@ void FormatNopcodeBytes(WORD nBaseAddress, DisasmLine_t& line_)
...
+			case NOP_STRING_APPLESOFT:
 				iByte = line_.nOpbyte;
 				strncpy(pDst, (const char*)(mem + nBaseAddress), iByte);
 				pDst += iByte;
 				*pDst = 0;
-			}
-			// Intentional fall through. // REVIEW??
-		case NOP_STRING_APPLE:
-			{
+				// Intentional fall through. // REVIEW??
```
Comment (kiyolee) [inline source/Debugger/Debugger_Disassembler.cpp:580]: @Michaelangel007 Is the fall through intentional?

File: source/Debugger/Debug.cpp [hunk 6]
```diff
@@ -5191,7 +5191,7 @@ Update_t CmdNTSC (int nArgs)
 						uint8_t *pTemp64x256 = new uint8_t[ 64 * 256 * 4 ];
 						memset( pTemp64x256, 0, g_nChromaSize );
 
-//Transpose16x1::transposeFrom16x1( pSwizzled, (uint8_t*) pChromaTable );
+						//Transpose16x1::transposeFrom16x1( pSwizzled, (uint8_t*) pChromaTable );
```
Comment (kiyolee) [inline source/Debugger/Debug.cpp:5194]: This is new.

File: source/Debugger/Debugger_Disassembler.cpp [hunk 7]
```diff
@@ -466,11 +468,10 @@ void FormatOpcodeBytes(WORD nBaseAddress, DisasmLine_t& line_)
 
 struct FAC_t
 {
-	uint8_t negative;
-	 int8_t exponent;
-	uint32_t mantissa;
-
-	bool     isZero;
+	uint8_t		negative;
```
[the remark this answers is not in the document]
Comment (kiyolee) [reply source/Debugger/Debugger_Disassembler.cpp]: Fixed in new commit.

File: source/Debugger/Debugger_Disassembler.cpp [hunk 8]
```diff
@@ -522,10 +523,11 @@ void FormatNopcodeBytes(WORD nBaseAddress, DisasmLine_t& line_)
 
 		switch (line_.iNoptype)
 		{
-			case NOP_BYTE_1:
-			case NOP_BYTE_2:
-			case NOP_BYTE_4:
-			case NOP_BYTE_8:
+		case NOP_BYTE_1:
```
[the remark this answers is not in the document]
Comment (kiyolee) [reply source/Debugger/Debugger_Disassembler.cpp]: Reverted in new commit.

File: source/Debugger/Debug.cpp (hunk 1 above)
Comment (kiyolee) [reply source/Debugger/Debug.cpp]: Reverted in new commit.

File: source/Debugger/Debug.cpp (hunk 2 above)
Comment (kiyolee) [reply source/Debugger/Debug.cpp]: Reverted in new commit.

File: source/Debugger/Debug.cpp (hunk 6 above)
Comment (Michaelangel007) [reply source/Debugger/Debug.cpp:5194]: Looks fine.

File: source/Debugger/Debug.cpp [hunk 9]
```diff
@@ -5251,7 +5251,7 @@ int CmdTextSave (int nArgs)
 	if (g_aArgs[1].bType & TYPE_QUOTED_2)
 		bHaveFileName = true;
 
-	char *pText;
+	char  *pText;
```
Comment (Michaelangel007) [inline source/Debugger/Debug.cpp:5254]: 👍

File: source/Debugger/Debugger_Disassembler.cpp [hunk 10]
```diff
@@ -468,10 +468,11 @@ void FormatOpcodeBytes(WORD nBaseAddress, DisasmLine_t& line_)
...
 {
-	uint8_t		negative;
-	int8_t		exponent;
-	uint32_t	mantissa;
-	bool		isZero;
+	uint8_t  negative;
+	 int8_t  exponent;
+	uint32_t mantissa;
+
+	bool     isZero;
```
Comment (Michaelangel007) [inline source/Debugger/Debugger_Disassembler.cpp:475]: Thanks for fixing this.

File: source/Debugger/Debugger_Disassembler.cpp (hunk 5 above)
Comment (Michaelangel007) [reply source/Debugger/Debugger_Disassembler.cpp:580]: I think that's a bug?  Thanks for the catch.  For now add a `break` until I setup a test case to exercise if the fall through is needed or not.

**Review by Michaelangel007 — APPROVED** [review]
Looks good. Thanks!

File: source/Debugger/Debug.cpp (hunk 2 above)
Comment (Michaelangel007) [reply source/Debugger/Debug.cpp]: 👍 Thanks!

**Review by Michaelangel007 — APPROVED** [review]
Thanks

**Comment by Michaelangel007** [comment]
@tomcw This PR is ready to be merged.  A quick set of eyes wouldn't hurt.

**Comment by tomcw** [comment]
LGTM - merged it.

**Comment by Michaelangel007** [comment]
@tomcw Thanks!

**Pull request merged**